Makefile

```diff
@@ -405,11 +405,13 @@ release:
 .PHONY: release-manual
 release-manual:
 	# this set is taken from esbuild's process https://github.com/evanw/esbuild/blob/master/Makefile#L330
-	@npm --version > /dev/null || (echo "The 'npm' command must be in your path to-*///// publish" && false)
+	@npm --version > /dev/null || (echo "The 'npm' command must be in your path to publish" && false)
 	@echo "Checking for uncommitted/untracked changes..." && test -z "`git status --porcelain | grep -vE ''`" || \
 		(echo "Refusing to publish with these uncommitted/untracked changes:" && \
 		git status --porcelain | grep -vE '' && false)
+	@echo "Checking that we're not on main branch..." && test main != "`git rev-parse --abbrev-ref HEAD`" || \
```

Comment:
Adding a check here since if you run `make release-manual` from main, you'll get into a bad state where npm has the new version but the rest of the script never finished.